src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -677,5 +677,18 @@ object messages {
            |
            |$fixedVarInAlternative""".stripMargin
     }
+  }  
+    
+  case class TypeParamsTypeExpected(mods:Modifiers, identifier:TermName)(implicit ctx:Context) extends Message(24) {
```

Comment:
We should always add a space between `:` and a type signature, e.g. `mods: Modifiers`